XR_APILAYER_NOVENDOR_toolkit/layer.cpp

```diff
@@ -583,8 +585,13 @@ namespace {
 
         void takeScreenshot(std::shared_ptr<graphics::ITexture> texture) const {
             std::stringstream parameters;
-            if (m_upscaleMode == config::ScalingType::NIS) {
-                parameters << "NIS_" << m_upscalingFactor << "_" << m_configManager->getValue(config::SettingSharpness);
+            if (m_upscaleMode != config::ScalingType::None) {
+                // TODO: add a getUpscaleModeName() helper to keep enum and string in sync.
+                const auto upscaleName = 
+                    m_upscaleMode == config::ScalingType::NIS ? "NIS_" :
+                    m_upscaleMode == config::ScalingType::FSR ? "FSR_" : "SCL_";
+
+                parameters << upscaleName << m_upscalingFactor << "_" << m_configManager->getValue(config::SettingSharpness);
```

Comment:
Sharpness could be omitted when not using NIS/FSR.

Reply:
If the intent is to help comparing the effect of these values in comparing screenshots, which you can differentiate by their file names, I'd keep both scaling mode and sharpness values still in the file name to make it easier to pick up the files when sorted alphabetically in File Explorer for example.

Reply:
Eventually both scale and sharpness could be used to format the file name even if they are default (removing the test for ScalingType::None) so that screenshots file names are all fitting the same template and size regardless. 

This is a design decision I let you decide!

Reply:
It's just a bit weird that right now we have `SCL_` with a sharpness value, which is the value from "whenever NIS or FSR was last enabled".

Reply:
After reconsidering this, there isn't any one size fits all format anyhow. Do you prefer limiting the use of the prefix only for the NIS and FSR cases?

Reply:
This is what I had originally yes. Let's change this in a separate commit.